pkg/util/pki/common.go

```diff
@@ -153,14 +153,14 @@ func clusterDNSNames(cluster *v1beta1.KafkaCluster) (names []string) {
 }
 
 // LabelsForKafkaPKI returns kubernetes labels for a PKI object
-func LabelsForKafkaPKI(name string) map[string]string {
-	return map[string]string{"app": "kafka", "kafka_issuer": fmt.Sprintf(BrokerIssuerTemplate, name)}
+func LabelsForKafkaPKI(name, namespace string) map[string]string {
+	return map[string]string{"app": "kafka", "kafka_issuer": fmt.Sprintf(BrokerClusterIssuerTemplate, name, namespace)}
```

Comment:
I'd change the order to `namespace, name`.

Reply:
Fixed.